### PR TITLE
Get SpecFlow tool version from csproj file

### PR DESCRIPTION
**Pull request by oakio**

### CODE REVIEW

File: src/SpecFlow.NetCore/Fixer.cs [hunk 1]
```diff
@@ -4,43 +4,31 @@
...
-			_specFlowExe = FindSpecFlow(specFlowPath);
-			WriteLine("Found: " + _specFlowExe);
-
+			_specFlowExe = specFlowPath;
 			_testFramework = testFramework;
 			_toolsVersion = toolsVersion;
 		}
 
-		private string FindSpecFlow(string path)
+		private static string FindSpecFlow(string version)
```
Comment (stajs) [inline src/SpecFlow.NetCore/Fixer.cs:27]: Reason for `static`?

File: src/SpecFlow.NetCore/Fixer.cs [hunk 2]
```diff
@@ -65,13 +53,32 @@ private string FindSpecFlow(string path)
 			throw new Exception($"Can't find SpecFlow: {path}\nTry specifying the path with {Args.SpecFlowPathArgName}.");
 		}
 
+		private static bool TryGetSpecFlowVersion(FileInfo csproj, out string version)
```
Comment (stajs) [inline src/SpecFlow.NetCore/Fixer.cs:56]: Again, reason for `static`?

**Review by stajs — CHANGES_REQUESTED** [review]
Thanks for the PR! Just the one question (repeated twice); I couldn't see a need for `static` methods unless I missed something...

File: src/SpecFlow.NetCore/Fixer.cs (hunk 1 above)
Comment (oakio) [reply src/SpecFlow.NetCore/Fixer.cs:27]: It's like a hint (for those who will read this code) that the method does not use class members.

File: src/SpecFlow.NetCore/Fixer.cs (hunk 2 above)
Comment (oakio) [reply src/SpecFlow.NetCore/Fixer.cs:56]: It's like a hint (for those who will read this code) that the method does not use class members.

**Comment by stajs** [comment]
https://www.nuget.org/packages/SpecFlow.NetCore/1.1.0

**Pull request merged**